lib/console/board_renderer.rb

```diff
@@ -0,0 +1,16 @@
+class BoardRenderer
+  def render(board)
+    tile_strings = board.tiles.map { |tile| tile.to_s }
```

Comment:
Ruby gives you this idiomatic shorthand for that kind of stuff: `board.tiles.map(&:to_s)`

https://www.brianstorti.com/understanding-ruby-idiom-map-with-symbol/

Reply:
nice!